scripts/prepare_alpaca_libre.py

```diff
@@ -0,0 +1,161 @@
+"""Implementation derived from https://github.com/tloen/alpaca-lora"""
+import json
+import sys
+from pathlib import Path
+
+import requests
+import torch
+from torch.utils.data import random_split
+from tqdm import tqdm
+
+# support running without installing as a package
+wd = Path(__file__).parent.parent.resolve()
+sys.path.append(str(wd))
+
+from lit_gpt.tokenizer import Tokenizer
+
+DATA_FILE_URL = "https://raw.githubusercontent.com/mobarski/alpaca-libre/main/data/output/alpaca_libre_ok_tasks_v4.json"
+DATA_FILE_NAME = "alpaca_libre_data_cleaned_archive.json"
+DESTINATION_PATH = Path("data/alpaca_libre")
```

Comment:
Is this worth adding an entirely new file? These are the only 3 changes.

We could just add a flag to `prepare_alpaca` or make alpaca libre the default

Reply:
Sounds good, I'll add a flag.